aldryn_events/tests/frontend/integration/specs/spec.add.delete.event.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global describe, it, browser */
+
+// #############################################################################
+// INTEGRATION TEST
+describe('Aldryn Events tests: ', function () {
+    var aldrynTestsLibrary = require('../pages/alteli.js');
+    var aldrynEventsPage = require('../pages/page.add.delete.event.js');
+
+    it('should log in to the site with valid username and ' +
+        'password', function () {
```

Comment:
You don't need this break, and it just makes the code harder and more confusing to read.